graphrag_eval/aggregation.py

```diff
@@ -88,7 +88,7 @@ def update_steps_summary(
         if step["status"] != "error":
             try:
                 res = json.loads(step["output"])
-                if "results" in res and "bindings" in res["results"]:
+                if isinstance(res, dict) and "results" in res.keys() and "bindings" in res["results"]:
```

Comment:
This is enough:
```suggestion
                if isinstance(res, dict) and "results" in res and "bindings" in res["results"]:
```
We do not need to specify `.keys()`.